Week07/lesson14/classwork/responsive_articles_with_menu/css/styles.css

```diff
@@ -0,0 +1,216 @@
+/* Box Model Hack */
+* {
+ box-sizing: border-box;
+}
+
+/* Clearfix hack */
+.clearfix:before, .clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after{
+  clear: both;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  font-size: 13px;
+  background-color: #F6F0E4;
+}
+
+/* Basic/Mobile Article Styling: */
+
+nav {
+  background-color: #7BAFA4;
+}
+
+nav article {
+  padding: 0 20px;
+}
+
+menu {
+  float: right;
+}
+
+.menubutton {
+  position: relative;
+  background: white;
+  height: 5px;
+  width: 30px;
+  border-radius: 9px;
+  margin-bottom: 2px;
+}
+
+ul {
+  padding: 0;
+  margin: 0 20px;
+  transition: max-height 0.5s;
+  max-height: 0px;
+  overflow: hidden;
+}
+
+.visible {
+  max-height: 606px;
+}
+
+li {
+  padding: 10px;
+  border-bottom: 1px solid #F6F0E4;
+}
+
+li:first-child {
+  border-top: 1px solid #F6F0E4;
+}
+
+li:last-child {
+  margin-bottom: 20px;
+}
+
+li:hover, li:focus, menu:hover > div, menu:focus > div {
```

Comment:
I would still need a list of 4 variations of pseudo elements for that .shared-class though, right? 

:hover
:focus
:hover > div
:focus > div